crates/ruma-common/CHANGELOG.md

```diff
@@ -23,6 +24,7 @@ Breaking changes:
   `deserialize_as_number_or_string` to reflect that.
 - The http crate had a major version bump to version 1.1
 - `IntoHttpError::Header` now contains a `HeaderSerializationError`
+- Use OpenAPI's syntax for path parameters in all metadata constants (?).
```

Comment:
I don't think "OpenAPI's syntax" is clear. Just say something like "use braces instead of colon for path parameters".

Also this should be under "unreleased".